portfolio/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Very nice work and nice structured code! Maybe you could have spent a little bit of time with the projects, contact and skills sections to make them stand out a bit.

Reply:
thank you! I will focus on the projects as well